spec/shared_selenium_session.rb

```diff
@@ -388,6 +388,31 @@
           @animation_session.find_link('animate me away').right_click
           expect(@animation_session).to have_content('Animation Ended', wait: 0.1)
         end
+
+        it 'should scroll the page instantly', requires: [:js] do
+          @animation_session.visit('with_animation')
```

Comment:
I think these tests would probably be more reliable if it was done all in one `evaluate_script`. Something like - ```scrollY = @animation_session.evaluate_script(<<~JS)
      (function(){
        window.scrollTo(0,500);
        return window.scrollY;
      })()
    JS```